src/components/toolbar/toolbar.spec.ts

```diff
@@ -20,45 +15,32 @@ describe('MdToolbar', () => {
     TestBed.compileComponents();
   }));
 
-  beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
-    builder = tcb;
-  }));
-
-  it('should apply class based on color attribute', (done: () => void) => {
-    return builder.createAsync(TestApp).then((fixture) => {
-      let testComponent = fixture.debugElement.componentInstance;
-      let toolbarDebugElement = fixture.debugElement.query(By.css('md-toolbar'));
-
-      testComponent.toolbarColor = 'primary';
-      fixture.detectChanges();
+  it('should apply class based on color attribute', () => {
+    let fixture = TestBed.createComponent(TestApp);
+    let testComponent = fixture.debugElement.componentInstance;
+    let toolbarDebugElement = fixture.debugElement.query(By.css('md-toolbar'));
```

Comment:
This is old from me - what do you think about using the `nativeElement` right here, instead of calling its getter each time?